feature-group-definitions/contain-intrinsic-size.dist.yml

```diff
@@ -0,0 +1,25 @@
+# Generated from: contain-intrinsic-size.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: contain-intrinsic-size
+spec: https://drafts.csswg.org/css-sizing-4/#intrinsic-size-override
+# Shown as "intrinsic-size" on chromestatus.com, but 651 is contain-intrinsic-size.
+usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/651
+status:
+  baseline: low
+  baseline_low_date: 2023-09-18
+  support:
+    chrome: "117"
```

Comment:
The important parts were supported much earlier. Need to review:
https://developer.mozilla.org/en-US/docs/Web/CSS/contain-intrinsic-size#browser_compatibility